DatabaseHandler/DatabaseController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DatabaseHandler
+{
+    public class DatabaseController
+    {
+        public Account GetAccountByUser(string username)
+        {           
+            using (Maplestory2DBEntities context = new Maplestory2DBEntities())
+            {
+                Account accountInfo = context.Accounts.FirstOrDefault(r => r.Username == username);
+                return accountInfo;
+            }
+        }
+
+        public void CreateChar(long AccID, string name, byte gender, int JobId, int MapId)
+        {
+            using (Maplestory2DBEntities context = new Maplestory2DBEntities())
```

Comment:
Do you need context in this method?

Reply:
Allows me to open a connection each time that method is called and trashes the connection when its done. So I guess?